piano-hero/core/src/main/java/com/novoda/pianohero/Sequence.java

```diff
@@ -0,0 +1,42 @@
+package com.novoda.pianohero;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+public final class Sequence {
+
+    private final List<Set<Note>> notes;
```

Comment:
I couldn't think of a name for `Set<Note>`. It's the column of notes you'd play together, potentially with both hands. I guess it's still technically a chord? But it could also be just a set of 1, so it could also not be a chord.